src/main/java/seedu/address/model/transaction/exceptions/PayeeIsPayerException.java

```diff
@@ -0,0 +1,8 @@
+//@@author steven-jia
+package seedu.address.model.transaction.exceptions;
+
+/**
+ * Signals that one of the specified payees is the payer.
+ */
+public class PayeeIsPayerException extends Exception {
```

Comment:
You creates this Exception, but never used it.

Reply:
Good point. I was planning to but found out I only needed CommandException. Will delete